### PR TITLE
Add jira project_keys field

### LINKED ISSUE #1959: Support new field: jira integration "project_keys"
In Gitlab 1.17.0, they have renamed `project_key` to `project_keys`

https://docs.gitlab.com/ee/api/integrations.html#jira

Discussion around Terraform provider --> https://gitlab.com/gitlab-org/terraform-provider-gitlab/-/issues/6303

### PR DESCRIPTION
**Pull request by ebuildy**
Add new field from gitlab v17

https://docs.gitlab.com/ee/api/integrations.html#jira

Capture d’écran, le 2024-06-23 à 19 03 32

fix #1959

### CODE REVIEW

File: services.go [hunk 1]
```diff
@@ -1171,16 +1174,17 @@ func (s *ServicesService) GetJiraService(pid interface{}, options ...RequestOpti
...
-	Username              *string `url:"username,omitempty" json:"username,omitempty" `
-	Password              *string `url:"password,omitempty" json:"password,omitempty" `
-	Active                *bool   `url:"active,omitempty" json:"active,omitempty"`
-	JiraIssueTransitionID *string `url:"jira_issue_transition_id,omitempty" json:"jira_issue_transition_id,omitempty"`
-	CommitEvents          *bool   `url:"commit_events,omitempty" json:"commit_events,omitempty"`
-	MergeRequestsEvents   *bool   `url:"merge_requests_events,omitempty" json:"merge_requests_events,omitempty"`
-	CommentOnEventEnabled *bool   `url:"comment_on_event_enabled,omitempty" json:"comment_on_event_enabled,omitempty"`
+	URL                   *string   `url:"url,omitempty" json:"url,omitempty"`
+	APIURL                *string   `url:"api_url,omitempty" json:"api_url,omitempty"`
+	ProjectKey            *string   `url:"project_key,omitempty" json:"project_key,omitempty" `
```
Comment (svanharmelen) [inline services.go:1179]: Guess this should also be moved to the bottom wit a deprecated comment...

**Review by svanharmelen — APPROVED** [review]
Code needed a format, but is OK now 👍🏻  Thanks!

**Comment by ebuildy** [comment]
Many thanks, when do you plan to make a new version please?

**Comment by svanharmelen** [comment]
Just released `v0.106.0`

**Pull request merged**